.github/workflows/codeql.yml

```
@@ -0,0 +1,100 @@
+      # required to fetch internal or private CodeQL packs
+      packages: read
+
+      # only required for workflows in private repositories
+      actions: read
+      contents: read
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - language: actions
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 8

(indentation)